tests/scripts/create-dev-cluster.sh

```diff
@@ -147,41 +173,56 @@ while getopts ":hrmfd:" opt; do
 	d)
 	    ROOK_EXAMPLES_DIR="$OPTARG"
 	    ;;
+    y)
+	    minikube_driver="$OPTARG"
```

Comment:
@BlaineEXE  wrote:

> Should `minikube_driver` be all caps to keep consistency with `ROOK_EXAMPLES_DIR` above?

In fact, I started with all caps but then changed it to lower case because I liked it better. I can still do the change if you want me to.

Reply:
Generally, I find that caps are used in scripts for variables that have a long-lasting scope, versus lower case for variables that have a shorter scope. That seems to be the trend followed by this script and ROOK_EXAMPLES_DIR. I think it is important to keep internal consistency where possible, regardless of personal preferences.